### ISSUE TITLE
Ability to filter out posts without Alt Text

### ISSUE THREAD

**Issue by RetroSunstar** [bluesky-social/social-app#7877]
### Describe the Feature

An in-app way to filter out posts that have images but no alt-text

Currently the only way to achieve this is by making a feed. It's clear there's a number of people who don't know about, forget, or simply don't care enough to put alt text on their images

Having a toggle in the base app to not show posts without alt text would be beneficial to people who rely on it, people who don't want to see or interact with posts that don't have it, and possibly help make images more accessible by proxy

Potential issue:

This could also encourage people to put meaningless text instead, to try and get it to that audience without putting in any work. A similar reason to why making the "Require Alt Text" toggle on by default was denied in #1099

A possible solution could be a muted terms function for specifically alt text, which also allows you to specify: "If this is the *entire* text, don't show it" particularly useful for alt text that's just the word "art" or "alt text" and nothing else, but without removing images that have those words in them

### Additional Context

Handling multiple images, sometimes posters will add alt text to the first image in their post but simply lack it on subsequent images, an option for this would also be nice to have.

While you could also achive this affect with a labeller as well, there's simply so many posts that don't contain alt text that it outweights the rate limits for labeller from what I'm aware???

**Comment by surfdude29** [comment]
Please see #4550 for discussion of this.
